apps-rendering/src/components/Layout/LetterLayout.tsx

```diff
@@ -60,54 +60,59 @@ interface Props {
 	item: LetterItem;
 }
 
-const LetterLayout: FC<Props> = ({ item }) => (
-	<main css={[Styles, DarkStyles]}>
-		<article css={BorderStyles}>
-			<header>
-				<Series item={item} />
-
-				<DesignTag format={item} />
+const LetterLayout: FC<Props> = ({ item }) => {
```

Comment:
The only change in this file is

the addition of WithAgeWarning component
adding const format = getFormat(item);
removal of DesignTag. DesignTag has been moved into the Headline